packages/main/src/plugin/color-registry.ts

```
@@ -418,256 +422,262 @@ export class ColorRegistry {
     });
 
     this.registerColor(`${sNav}expander`, {
-      dark: white,
-      light: charcoal[700],
+      dark: '#e8eef7',
+      light: navy[400],
     });
   }
 
   protected initCardContent(): void {
     this.registerColor(`card-bg`, {
```

Biome (2.4.13)

[error] Do not use template literals if interpolation and special-character handling are not needed.

(lint/style/noUnusedTemplateLiteral)